tests/Laravel/ServiceProviderTest.php

```diff
@@ -113,15 +113,24 @@ public function testListenerWithoutLocaleUpdatedClass()
         CarbonImmutable::setLocale('en');
         CarbonPeriod::setLocale('en');
         CarbonInterval::setLocale('en');
+        Carbon::setFallbackLocale('en');
+        CarbonImmutable::setFallbackLocale('en');
+        CarbonPeriod::setFallbackLocale('en');
+        CarbonInterval::setFallbackLocale('en');
 
         $service->boot();
         $service->app->register();
         $service->app->setLocaleWithoutEvent('fr');
+        $service->app->setFallbackLocale('it');
         $dispatcher->dispatch('locale.changed');
         $this->assertSame('fr', Carbon::getLocale());
         $this->assertSame('fr', CarbonImmutable::getLocale());
         $this->assertSame('fr', CarbonPeriod::getLocale());
         $this->assertSame('fr', CarbonInterval::getLocale());
+        $this->assertSame('en', Carbon::getFallbackLocale());
+        $this->assertSame('en', CarbonImmutable::getFallbackLocale());
+        $this->assertSame('en', CarbonPeriod::getFallbackLocale());
+        $this->assertSame('en', CarbonInterval::getFallbackLocale());
```

Comment:
I actually test here that event NOT having effect on fallback locale. Dunno if it makes any sense😅 /1